src/functions/authoriser/framework/handler.ts

```diff
@@ -47,10 +47,10 @@ function createAuthResult(
   principalId: string, effect: Effect, resource: string,
 ): CustomAuthorizerResult {
   const staffNumber = Array.isArray(employeeId) ? employeeId[0] : employeeId;
-  const role = getEmployeeRole(employeeId);
+  const examinerRole = getExaminerRole(employeeId);
```

Comment:
Feels like this is the correct place to change?

Reply:
Done